### DISCUSSION TITLE
Include Check box in webmail for select all messages

### DISCUSSION THREAD

**Discussion by danilofavacho** [modoboa/modoboa#2111] [Ideas]
Hello guys, I'm a new user of modoboa and I'm learning to configure and use the environment. I'm having trouble deleting several emails from the inbox due to the lack of a checkbox at the top that would allow us to select all messages. Is this a bug or is it an improvement to be made in the system? I thank everyone.

**Comment by tonioo** [comment]
You can select multiple messages and then drag&drop them to the trash folder.

**Reply by reesing322** [reply]
Correct, but wouldn't it be a helpful feature to have a 'select all' checkbox just above the list?

**Reply by shotcollin** [reply]
A 'select all' checkbox would be a really useful feature. 

Clicking checkboxes one a time is pretty tedious if wanting to select more than a dozen or so. And for hundreds or more, not really an option unless you've got an entire afternoon to waste. And it doesn't appear you can select one message and then Shift + select one further below it to select everything between either, unless I'm missing something? 

Being able to use either bulk-select method would be terribly convenient. Having both would be positively thrilling.
